app/src/App.tsx

```diff
@@ -46,10 +47,11 @@ export const App = () => {
 
   const { isInitializing, isAuthenticated, user } = useAuth0();
 
-  return (
-    <React.Fragment>
-
+  const theme = { mode: 'dark' }
```

Comment:
seeing the 'dark mode' theme here - is this synonymous to passing in an organization for example? or an additional styling layer?